test/runners/c_spec.js

```diff
@@ -1,77 +1,169 @@
 var expect = require('chai').expect;
 var runner = require('../runner');
 
-describe( 'c runner', function(){
-    describe( '.run', function(){
-        it( 'should handle basic code evaluation', function(done){
-            var solution = ['#include <stdio.h>',
-                'int main() {',
-                '    printf("I think the next language codewars should support is PL/I.  Wikipedia says it\'s still used in production to this day; presumably in nursing homes?");',
-                '}',
-                ''].join('\n');
+describe('c runner', function() {
+describe('.run', function() {
```

Comment:
Also don't forget to add examples to `examples/c.yml` and add `runner.assertCodeExamples('c');` here, so that those examples get tested. 

The examples are used later within all of our products as example code for demonstrating how to use the language.